app/server/src/features/accounts/methods.unit.test.ts

```diff
@@ -0,0 +1,243 @@
+    test('should find a single org', async () => {
+        const user = {
+            ...userData,
+            memberOf: [
+                {
+                    organization: 'Prytaneum',
+                },
+            ],
+        };
+
+        prismaMock.user.findUnique.mockResolvedValueOnce(user);
+
+        const expectedOutput = ['Prytaneum'];
+
+        const output = await AccountMethods.findOrgsByUserId(userData.id, prismaMock);
+
+        expect(output).toEqual(expectedOutput);
+    });
+
+    test('should find multiple orgs', async () => {
+        const user = {
+            ...userData,
+            memberOf: [
+                {
+                    organization: 'Prytaneum',
+                },
+                {
+                    organization: 'University of California, Riverside',
+                },
+                {
+                    organization: 'School of Public Policy',
+                },
+                {
+                    organization: 'Southern California',
+                },
+            ],
+        };
+
+        prismaMock.user.findUnique.mockResolvedValueOnce(user);
+
+        const expectedOutput = [
+            'Prytaneum',
+            'University of California, Riverside',
+            'School of Public Policy',
+            'Southern California',
+        ];
+
+        const output = await AccountMethods.findOrgsByUserId(userData.id, prismaMock);
+
+        expect(output).toEqual(expectedOutput);
+    });
+
+    test('should not find orgs', async () => {
+        prismaMock.user.findUnique.mockRejectedValueOnce(null);
+
+        await expect(AccountMethods.findOrgsByUserId(userData.id, prismaMock)).rejects.toBeNull();
+    });
+});
+
+/*
+ *  test cases for the function called when a user is registering themselves
+ */
+describe('registerSelf', () => {
+    test('should register a user if passwords match', async () => {
+        const input = {
+            email: userData.email,
+            firstName: userData.firstName,
+            lastName: userData.lastName,
+            password: userTextPassword,
+            confirmPassword: userTextPassword,
+        };
+
+        // mock registered user
+        const registeredUser = {
+            ...userData,
+        };
+
+        prismaMock.user.create.mockResolvedValueOnce(registeredUser);
+
+        // get jwt token to compare
+        const token = await jwt.sign({ id: toUserId(registeredUser).id });
+
+        const expectedOutput = { registeredUser, token };
+
+        const output = await AccountMethods.registerSelf(prismaMock, input);
+
+        expect(output).toEqual(expectedOutput);
+    });
+
+    test('should throw if passwords do not match', async () => {
+        const input = {
+            email: userData.email,
+            firstName: userData.firstName,
+            lastName: userData.lastName,
+            password: faker.internet.password(),
+            confirmPassword: faker.internet.password() + '#',
```

Comment:
Two different calls here would already generate two different passwords?  That is, if we even keep using faker per my other comment.

Reply:
I just wasn't sure if it was guaranteed that the passwords would always be different. There could be a case even if it's a slim possibility but I figured adding a character would prevent that.

Reply:
Makes sense